src/NexusMods.App/Program.cs

```diff
@@ -43,6 +43,9 @@ public static int Main(string[] args)
             loggingSettings
         );
 
+        // Okay to do wait here, as we are in the main process thread.
+        host.StartAsync().Wait();
```

Comment:
```suggestion
        // Okay to do wait here, as we are in the main process thread.
        host.StartAsync().Wait(timeout: TimeSpan.FromMinutes(5));
```

Timeout so we don't wait forever.